scripts/scripts/vaccinations/automations/incremental/faeroe_islands.py

```diff
@@ -1,31 +1,59 @@
-import json
 import re
 import requests
 import dateparser
 import vaxutils
+import pandas as pd
 
 
-def main():
+def read(source: str) -> pd.DataFrame:
+    return pd.read_json(requests.get(source).content)
```

Comment:
This should work:

```suggestion
    return pd.read_json(source)
```

Reply:
Then you could even remove the dependency on `requests`.

Reply:
The JSON we pull is like this:

```json
	{
	"stats": [
		{
			"last_updated" : "11. februar kl. 11:21",
			"cases" : "657",
			"active_cases" : "4", 
			"recovered" : "652",
			"deaths" : "1",
			"quarantined" : "19",
			"tested" : "225.960",
			"awaiting_results" : "1",
			"out_of_quarantine" : "",
			"hospitalized" : "1",
				"first_vaccine_number" : "4134",
	"first_vaccine_procentage" : "7.8",
	"second_vaccine_number" : "1221",
	"second_vaccine_procentage" : "2.3",
	"vaccine_last_update" : "10. februar kl. 09:59"
			}
		]
	}
```

By piping this straight into a data frame, we get a data frame of one row and one column (`stats`).

It would be better to pipe this as early as possible into something that’s easier to consume, e.g.:


```python
    return pd.read_json(requests.get(source).content).pipe(lambda ds: pd.DataFrame.from_records(ds["stats"]).iloc[0])
```

I made it an anonymous function here, it should be another step but the idea is there.
Incidentally, this would make the function return a Series and not a DataFrame.

This would return a series like:

```
last_updated                 11. februar kl. 11:21
cases                                          657
active_cases                                     4
recovered                                      652
deaths                                           1
quarantined                                     19
tested                                     225.960
awaiting_results                                 1
out_of_quarantine
hospitalized                                     1
first_vaccine_number                          4134
first_vaccine_procentage                       7.8
second_vaccine_number                         1221
second_vaccine_procentage                      2.3
vaccine_last_update          10. februar kl. 09:59
Name: 0, dtype: object
```

Reply:
This would make downstream manipulations more straightforward rather that accessing `["stats"][0]` all the time.